src/server/src/routes/job.ts

```diff
@@ -59,23 +59,6 @@ router.post('/create', async (req: jobRequest, res: Response) => {
         });
     }
     try {
-        // Check if required field is missing.
```

Comment:
Why is this being deleted

Reply:
There is exactly the same thing a few lines above.

Reply:
oh lol